src/cli.js

```diff
@@ -196,7 +197,15 @@ program
             }),
       });
 
-      if (documents.count === 0) {
+      const htmlFiles = [...documents.iter({pathOnly: true})].filter(filePath => filePath.endsWith(HTML_FILENAME));
+
+      function* fileIter() {
+        for (const filePath of htmlFiles) {
+          yield Document.read(filePath);
+        }
+      }
```

Comment:
If we're filtering the files out ourselves, let's skip creating a generator function and just iterate `htmlFiles` directly.

```suggestion
```